deploy/function_export_mv_to_table.sql

```diff
@@ -0,0 +1,461 @@
+          and address.type = 'Organisation';
+      alter table ggircs.address add constraint ggircs_address_organisation_foreign_key foreign key (organisation_id) references ggircs.organisation(id);
+
+      -- Create FK/PK relation between Address and parent_organisation
+      alter table ggircs.address add column parent_organisation_id int;
+      create index ggircs_address_parent_organisation_index on ggircs.address (parent_organisation_id);
+      update ggircs.address set parent_organisation_id = parent_organisation.id from ggircs.parent_organisation
+        where address.ghgr_import_id = parent_organisation.ghgr_import_id
+      and address.type = 'ParentOrganisation'
+      and address.parent_organisation_idx = parent_organisation.parent_organisation_idx;
+      alter table ggircs.address add constraint ggircs_address_parent_organisation_foreign_key foreign key (parent_organisation_id) references ggircs.parent_organisation(id);
+
+      -- Create FK/PK relation between Contact and Address
+      alter table ggircs.contact add column address_id int;
+      create index ggircs_contact_address_index on ggircs.contact (address_id);
+      update ggircs.contact set address_id = address.id from ggircs.address
+          where address.ghgr_import_id = contact.ghgr_import_id
+          and   address.type = 'Contact'
+          and   address.contact_idx = contact.contact_idx
+      ;
+      alter table ggircs.contact add constraint ggircs_contact_address_foreign_key foreign key (address_id) references ggircs.address(id);
+
+      -- Create FK/PK relation between Descriptor and Activity
+      alter table ggircs.descriptor add column activity_id int;
+      create index ggircs_descriptor_activity_index on ggircs.descriptor (activity_id);
+      update ggircs.descriptor set activity_id = activity.id from ggircs.activity
+          where descriptor.ghgr_import_id = activity.ghgr_import_id
+            and descriptor.process_idx = activity.process_idx
+            and descriptor.sub_process_idx = activity.sub_process_idx
+            and descriptor.activity_name = activity.activity_name;
+      alter table ggircs.descriptor add constraint ggircs_descriptor_activity_foreign_key foreign key (activity_id) references ggircs.activity(id);
+
+      -- Create FK/PK relation between Fuel and Unit
+      alter table ggircs.fuel add column unit_id int;
+      create index ggircs_fuel_unit_index on ggircs.fuel (unit_id);
+      update ggircs.fuel set unit_id = unit.id from ggircs.unit
+          where fuel.ghgr_import_id = unit.ghgr_import_id and fuel.process_idx = unit.process_idx  and fuel.sub_process_idx = unit.sub_process_idx
+            and fuel.activity_name = unit.activity_name and fuel.units_idx = unit.units_idx and fuel.unit_idx = unit.unit_idx;
+      alter table ggircs.fuel add constraint ggircs_fuel_unit_foreign_key foreign key (unit_id) references ggircs.unit(id);
+
+    -- Create FK/PK relation between Organisation and Parent Organisation
```

Comment:
Is this relation backwards?

Reply:
it's a many to many relationship. So we will need FKs on both...